app/components/overview-info-form/component.js

```diff
@@ -28,61 +30,69 @@ var generateValidators = function(questions) {
 
 const questions = {
   q1: {
-    question: 'Age',
+    question: translations.survey.sections['1'].questions['1'].label,
```

Comment:
As long as you don't have any need to iterate over sections that's fine. It might be easier to work with if the structure is consistent, but I'll leave that up to you.